bin/local-env/env-check.sh

```diff
@@ -7,7 +7,7 @@ set -u
 . "$(dirname "$0")/includes.sh"
 
 # If the WordPress container isn't running, start the environment.
-if ! dc ps -q $CONTAINER ; then
+if ! container ls > /dev/null ; then
```

Comment:
This isn't directly related but fixes the script which hasn't worked for a while.